### ISSUE TITLE
More documentation on using local (non-zoo) bundles with MONAILabel

### ISSUE THREAD

**Issue by wyli** [Project-MONAI/MONAILabel#1490]
bring your own bundle from local directories

**Comment by ericspod** [comment]
This is an important one I think for people developing bundles and wanting to experiment/visualise with their results. My solution to the problem was to download the bundle app and then soft link the bundle's directory into the directory where the app would normally download bundles from the zoo. This maybe wasn't correct but I could find documentation elsewhere about how to do things with local bundles. 

My script for doing this which assumes it's in the `docs` directory of a bundle:

```sh
#! /bin/bash

eval "$(conda shell.bash hook)"
conda activate monailabel

homedir="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"

BUNDLE="$(cd "$homedir/.." && pwd)"
BUNDLE_NAME="$(basename $BUNDLE)"
LABELDIR="$BUNDLE/monailabel"

if [ ! -d "$LABELDIR" ]
then
    mkdir "$LABELDIR"
    mkdir "$LABELDIR/datasets"
    cd "$LABELDIR"
    monailabel apps --download --name monaibundle
    mkdir "$LABELDIR/monaibundle/model"
    cd "$LABELDIR/monaibundle/model"
    ln -s "$BUNDLE" "$BUNDLE_NAME"
fi
    
cd "$LABELDIR"
monailabel start_server --app monaibundle --studies datasets --conf models "$BUNDLE_NAME" $*
```

**Comment by SachidanandAlle** [comment]
Correct.  Importing your own bundle being supported couple of versions (may be from 0.5 onwards).
You can dump your bundle into `model` folder and use the same instead of importing it from zoo.

Documentation on this is missing. We never highlighted this feature/flexibility.
@diazandr3s / @tangy5 please update the doc for this feature

Possibly add a section: `Bring Your Own Bundle` in
https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/monaibundle

**Comment by ericspod** [comment]
If possible it would be nice to have a command-line mechanism for making this work with `monailabel start_server` so that I can specify the bundle by path and a symlink is created automatically.
